contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -436,4 +436,25 @@ CREATE VIEW information_schema_tsql.check_constraints AS
 
 GRANT SELECT ON information_schema_tsql.check_constraints TO PUBLIC;
 
+/*
+* COLUMN_DOMAIN_USAGE
+*/
+
+CREATE OR REPLACE VIEW information_schema_tsql.COLUMN_DOMAIN_USAGE AS
+SELECT isc_col."DOMAIN_CATALOG",
```

Comment:
Fix indentation of the view. Leave a tab before SELECT, FROM and WHERE.

Reply:
done